go/apps/api/openapi/spec/paths/v2/ratelimit/multiLimit/V2RatelimitMultiLimitResponseBody.yaml

```diff
@@ -0,0 +1,12 @@
+type: object
+required:
+  - meta
+  - data
+properties:
+  meta:
+    "$ref": "../../../../common/Meta.yaml"
+  data:
+    type: array
+    description: Array of rate limit check results, one for each rate limit check in the request
+    items:
+      "$ref": "./V2RatelimitMultiLimitResponseData.yaml"
```

Comment:
my other idea would be to make this an object 

```
{
 "success": true/false -> indicating whether all limits passed or not as a quick indicator to use,
"limits" : [] same as the multiresponse
}
```

otherwise the caller of the api has to loop trough all arrays and do it themselves